.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "node-style-guide"
+  ],
+  "rules": {
+    indent: [2, 2],
+    "max-statements": [2, 50],
+    "new-cap": 0,
+    "no-eq-null": 2,
+    "no-unused-vars": [2, {"vars": "all", "args": "after-used"}],
+    "space-in-brackets": 0,
```

Comment:
Had to override this "node-style-guide" rule since it was causing all (9x) these errors:
```
/Users/pdehaan/dev/github/gulp/test/watch.js
  1:1  error  Rule 'space-in-brackets' was removed and replaced by: object-curly-spacing, array-bracket-spacing, computed-property-spacing  space-in-brackets
```

See https://github.com/felixge/node-style-guide/pull/71/files